client/my-sites/marketplace/components/reviews-cards/index.tsx

```diff
@@ -46,7 +48,7 @@ export const MarketplaceReviewsCards = ( props: ProductProps ) => {
 						className="is-link"
 						borderless
 						primary
-						onClick={ () => alert( 'Not implemented yet!' ) }
+						onClick={ () => props.showMarketplaceReviews && props.showMarketplaceReviews() }
```

Comment:
Can we update the button to use the same callback in the PluginDetailsHeader here:
https://github.com/Automattic/wp-calypso/blob/0c81f27d2f0858d8973d7f0564a8327235bfe1ed/client/my-sites/plugins/plugin-details-header/index.jsx#L93-L104

Or should this be handled in a separate PR?

Reply:
I would say this should come on the PRs that change the Plugin Details page layout.